unlock-app/src/services/storageService.js

```diff
@@ -0,0 +1,36 @@
+import axios from 'axios'
```

Comment:
We should probably use fetch, it's baked into every browser, and that includes in my PR for checking currency conversion rate. Low priority though

Reply:
This was influenced based on the design of the currencyConversionMiddleware, once this has landed and been merged in we can refactor this to use an alternate http library